File: URI/Beginner/Python/1010_SimpleCalculate.py
#=====================================    
#   URI Online Judge Problem
#   Category: Begginer
#   Problem: 1010 - Simple Calculate
#   Status: Accepted
#=====================================

input1 = input()
input2 = input()
input1 = input1.split()
input2 = input2.split()


number1 = int(input1[1])
priceUnit1 = float(input1[2])

number2 = int(input2[1])
priceUnit2 = float(input2[2])

print("VALOR A PAGAR: R$ {:.2f}".format(number1*priceUnit1 + number2*priceUnit2))

File: URI/Beginner/C/1010_SimpleCalculate.c
/*
	URI Online Judge Problem
	Category: Begginer
	Problem: 1010 - Simple Calculate
	Status: Accepted
*/

#include <stdio.h>

int main() {
	int id1, id2, amount1, amount2;
	float unitPrice1, unitPrice2;
	
	scanf("%d %d %f", &id1, &amount1, &unitPrice1);
	scanf("%d %d %f", &id2, &amount2, &unitPrice2);
	
	printf("VALOR A PAGAR: R$ %.2f\n", amount1*unitPrice1 + amount2*unitPrice2);
	
	return 0;
}
